docs/openapi.yaml

```
@@ -0,0 +1,1136 @@
+openapi: 3.1.0
+info:
+  title: Igloo Server API
+  description: |
+    A server-based signing device and personal ephemeral relay for the FROSTR protocol. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)